ansible/README.md

```diff
@@ -14,23 +14,23 @@ can be real hardware, VMs, things in a public cloud, etc. Anything that you can
 
 ### Configure inventory
 
-Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster.
+Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster. Place the 'inventory' file into the `./inventory` directory.
 
 ### Configure Cluster options
 
-Look though all of the options in `group_vars/all.yml` and
+Look though all of the options in `inventory/group_vars/all.yml` and
 set the variables to reflect your needs. The options are described there
 in full detail.
 
 ## Running the playbook
 
-After going through the setup, run the setup script provided:
+After going through the setup, run the deploy-cluster script provided by going into the scripts directory:
 
-`$ ./setup.sh`
+`$ cd scripts/ && ./deploy-cluster.sh`
 
 You may override the inventory file by doing:
 
-`INVENTORY=myinventory ./setup.sh`
+`INVENTORY=myinventory ./deploy-cluster.sh`
 
 
 In general this will work on very recent Fedora, rawhide or F21.  Future work to
```

Comment:
That is quite outdated. The README needs to be extended and polished. All three OS are already supported, including CoreOS. Subject of this PR is not to fix README, rather to establish new top directories structure. I plan to update all playbooks/documents and introduce new ones. Let's postpone the fix until we have improved framework including CI.